prow/jobs/kyma/kyma-gke-external-registry.yaml

```diff
@@ -0,0 +1,76 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+# Kubectl 1.15 and CLUSTER_VERSION=1.15
+base_image_k15: &base_image_k15
+  image: eu.gcr.io/kyma-project/test-infra/kyma-integration:v20200513-93bafa15-k8s1.15
+
+gke_job_template: &gke_job_template
+  optional: true
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+      - <<: *base_image_k15
+        securityContext:
+          privileged: true
+        command:
+          - "bash"
+        env:
+          - name: CLOUDSDK_COMPUTE_ZONE
+            value: "europe-west4-b"
+        args:
+          - "-c"
+          - "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-external-registry.sh"
+        resources:
+          requests:
+            memory: 200Mi
+            cpu: 80m
+
+gke_job_labels_template: &gke_job_labels_template
+  preset-kyma-guard-bot-github-token: "true"
+  preset-sa-gke-kyma-integration: "true"
+  preset-gc-compute-envs: "true"
+  preset-gc-project-env: "true"
+  preset-sa-test-gcr-push: "true"
+  preset-docker-push-repository-gke-integration: "true"
+  preset-dind-enabled: "true"
+  preset-kyma-artifacts-bucket: "true"
+  preset-cluster-use-ssd: "true"
+
+presubmits: # runs on PRs
+  kyma-project/kyma:
+    - name: pre-master-kyma-gke-external-registry
+      branches:
+        - ^master$
+      <<: *gke_job_template
+      # following regexp won't start build if only Markdown files were changed
+      run_if_changed: "^((resources\\\\serverless\\S+|installation\\S+|tools/kyma-installer\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
+      labels:
+        preset-build-pr: "true"
+        <<: *gke_job_labels_template
+      extra_refs:
+        - <<: *test_infra_ref
+          base_ref: master
+
+# postsubmits:
```

Comment:
Will it be used in future? If not then please remove the commented code

Reply:
It will be uncommented when the functionality will be merged to master...